doc/sphinx/Pacemaker_Explained/reusing-configuration.rst

```diff
@@ -262,10 +262,16 @@ Then instead of duplicating the rule for all your other resources, you can inste
 .. important::
 
    The cluster will insist that the ``rule`` exists somewhere.  Attempting
-   to add a reference to a non-existing rule will cause a validation
-   failure, as will attempting to remove a ``rule`` that is referenced
+   to add a reference to a nonexistent ``id`` will cause a validation failure,
+   as will attempting to remove a ``rule`` with an ``id`` that is referenced
    elsewhere.
 
+   Some rule syntax is allowed only in
+   :ref:`certain contexts <rule_conditions>`. Validation cannot ensure that the
```

Comment:
Currently true, probably.

It probably wouldn't be too complicated to expand IDREFs and then do another pass of schema validation. (If someone were comfortable with XSLT and with Pacemaker's schema code.) I think that would be a good wishlist item if there isn't one.

Reply:
https://projects.clusterlabs.org/T803